src/java/org/apache/cassandra/utils/JMXServerUtils.java

```diff
@@ -362,5 +365,24 @@ public String[] list() throws RemoteException {
         public void setRemoteServerStub(Remote remoteServerStub) {
             this.remoteServerStub = remoteServerStub;
         }
+
+        /**
+         * Closes the underlying JMX registry by unexporting this instance.
+         * There is no reason to do this except for in-jvm dtests where we need
+         * to stop the registry, so we can start with a clean slate for future cluster
+         * builds, and the superclass never expects to be shut down and therefore doesn't
+         * handle this edge case at all.
+         */
+        @VisibleForTesting
+        public void close() {
+            try
+            {
+                UnicastRemoteObject.unexportObject(this, true);
+            }
+            catch (NoSuchObjectException e)
```

Comment:
maybe rename the variable to indicate it's ignored
```suggestion
            catch (NoSuchObjectException ignored)
```